test/libsolidity/astPropertyTests/forLoop/isSimpleCounterLoop_negative_cases.sol

```diff
@@ -136,3 +161,7 @@ contract C {
 // StateVarCounterModifiedFunctionConditionRHS: false
 // StateVarCounterModifiedFunctionLoopBody: false
 // NonIntegerCounter: false
+// AssemblyAfterAssignmentToCounter: false
+// ExpressionIncrement: false
+// StructMemberAccessIncrement: false
+// DynamicArrayLengthConditionRHS: false
```

Comment:
Tests are failing because this AST check is not defiend. I pushed a fixup to remove the line.